specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -2387,6 +2387,97 @@
                     }
                 }
             }
+        },
+        "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/policy/latest": {
```

Comment:
To adhere to convention, this should be "policies" instead of "policy." Also, the convention for singleton resources is to use /default (as in {}/policy/default). Is there a specific reason to use /latest? Are policies other than the latest enumerable?

Reply:
Sure, changed from latest to default. 
There was some previous ARM feedback to use latest so thought of using it. That scenario is little different though.

Reply:
As of now policies are always latest.